test/framework/src/main/java/org/opensearch/test/OpenSearchTestCase.java

```diff
@@ -257,6 +258,7 @@ public void append(LogEvent event) {
         }));
 
         BootstrapForTesting.ensureInitialized();
+        TransportService.ensureClassloaded(); // ensure server streamables are registered
```

Comment:
I think you could bake the necessary checks for `TransportService` into `BootstrapForTesting.ensureInitialized();`

Reply:
For this PR it's 6 and one half. I opted this route so that I could remove later w/o any impact to `BootstrapForTesting`.